subjects/models.py

```diff
@@ -4,11 +4,53 @@
 from django.contrib.postgres.fields import ArrayField, JSONField
 from datetime import datetime, timezone
 
+class Subject(models.Model):
+    """Metadata about an experimental subject (animal or human)."""
+    SEXES = (
+    	('M', 'Male'),
+    	('F', 'Female'),
+    	('U', 'Unknown')
+    )
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    nickname = models.CharField(max_length=255, null=True, blank=True, unique=True,
+                                help_text="Easy-to-remember, unique name (e.g. “Hercules”).")
+    species = models.ForeignKey(Species)
+    litter = models.ForeignKey('Litter', null=True, blank=True)
+    sex = models.CharField(max_length=1, choices=SEXES, default='U')
+    strain = models.ForeignKey(Strain, null=True, blank=True)
+    #genotype = models.ForeignKey(Genotype, null=True, blank=True)
+	genotype = models.ManyToManyField(Allele) # NOTE: CAN WE ADD ANOTHER FIELD "Zygosity" TO THE JUNCTION TABLE?
```

Comment:
They are most commonly called "homozygous" and "heterozygous". Which unfortunately both begin with H.

So I guess an integer field is what we want...